pep-0606.rst

```diff
@@ -0,0 +1,223 @@
+sometimes skip releases due to the mismatch between their 24 month release
+cycles and CPython's typically 18-month cycle).
+
+The faster 12-month full release cadence in PEP 602 means that users in this
+category may end up skipping two releases where they would previously have only
+skipped one. However, the extended notice period for deprecations means that
+skipping a single release should no longer result in missed deprecation warnings.
+
+With a 12-month release cadence each release branch will spend proportionally
+more time in security-fix-only mode (~48 months out of ~60 vs the current ~42
+out of ~60).
+
+The slower 24-month full release cadence in PEP 605 may move some of the users
+that have historically been in this category into the "update to every stable
+release" category.
+
+With a 24-month release cadence each release branch will spend proportionally
+less time in security-fix-only mode (~36 months out of ~60 vs the current ~42
+out of ~60).
+
+
+Impact on users and redistributors that update to every release
+---------------------------------------------------------------
+
+Many of Python's users never install a pre-release, but do update to every
+stable release series at some point after it is published.
+
+PEP 602 aims to mitigate the potential negative impact on members of this group
+by keeping the minimum gap between releases to 12 months.
+
+PEP 605 aims to mitigates the potential negative impact on members of this group
+by focusing its change proposals on the management of the pre-release period.
+However, some users of the stable releases are likely to be unhappy with the
+slower release cadence in that PEP, while also being unwilling to make the jump
+to using the rolling beta release stream.
+
+Full discussion of the impact on this group is left to the individual PEPs.
+
+
+Impact on users and redistributors of CPython nightly builds
+------------------------------------------------------------
+
+Despite the difficulties of doing so, there are already some users and
+redistributors that take on the challenge of using or publishing the CPython
+master branch directly.
+
+Neither PEP 602 nor PEP 605 should directly affect this group, but the rolling
+release stream proposal in PEP 605 aims to substantially reduce the barriers to
+more users adopting this style of usage (except with the rolling beta stream,
+rather than using the master branch directly).
+
+
+Impact on maintainers of third party libraries
+----------------------------------------------
+
+For maintainers of third party libraries, the key source of support complexity
+is the *number* of different Python versions in widespread use.
+
+PEP 602 aims to mitigate the potential negative impact on members of this group
+by keeping the minimum gap between full releases to 12 months.
+
+PEP 605 aims to mitigate the potential negative impact on members of this group
+by increasing the gap between full releases to 24 months, and retaining the
```

Comment:
"... and reducing the number of simultaneous actively maintained full releases at any time to one ..."

Reply:
Oh, there's  a text change in https://www.python.org/dev/peps/pep-0602/#one-year-of-full-support-four-more-years-of-security-fixes that hasn't been reflected in the header yet (the support period went back up to 18 months)

Reply:
Fixed a few parts of PEP 606 that were incorrectly based on the title of that section, rather than the amended text of the bullet points.